docs/content/hive-getting-started.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Hive and Iceberg Quickstart"
+weight: 100
+url: hive-quickstart
+aliases:
+    - "quickstart"
+    - "quickstarts"
+    - "getting-started"
+disableSidebar: true
+disableToc: true
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+<!-- {{% quickstarts %}} -->
+
+## Hive and Iceberg Quickstart
+
+This guide will get you up and running with an Iceberg and Hive environment, including sample code to
+highlight some powerful features. You can learn more about Iceberg's Hive runtime by checking out the [Hive](../docs/latest/hive/) section.
+
+- [Feature-support](#feature-support)
+- [Enabling Iceberg support in Hive](#enabling-iceberg-support-in-hive)
+- [Catalog Management](#catalog-management)
+- [DDL Commands](#ddl-commands)
+- [DML Commands](#dml-commands)
+
+### Docker images
+
+The fastest way to get started is to use [Apache Hive images](https://hub.docker.com/r/apache/hive) 
+which provides a SQL-like interface to create and query Iceberg tables from your laptop. You need to install the [Docker Desktop](https://www.docker.com/products/docker-desktop/), choosing the Intel or Apple M1 chip if you have a Mac, or choosing Linux.
+
+Set the version variable:
+```export HIVE_VERSION=4.0.0-alpha-2```
+
+Start the container, using the option --platform linux/amd64 for a Mac M1:
+```docker run -d --platform linux/amd64 -p 10000:10000 -p 10002:10002 --env SERVICE_NAME=hiveserver2 --name hive4 apache/hive:${HIVE_VERSION}
+```
+
+This command configures Hive to use the embedded derby database for Hive Metastore. Hive Metastore functions as the Iceberg catalog to locate Iceberg files, which can be anywhere. 
```

Comment:
This line is referring to the previous command correct?

Reply:
Thanks. Fixing:
The linux/amd64 platform option configures Hive ...

Reply:
![image](https://github.com/apache/iceberg-docs/assets/16125343/3a7f4213-932e-4674-8a09-565348f5ec75)